### ISSUE TITLE
Add support for Tailwind 2.0

### ISSUE THREAD

**Issue by Uninen** [forsartis/vue-cli-plugin-tailwind#24]
Tailwind 2 has been released. Would be nice to be able to use it with this plugin as well 👍

**Comment by bodograumann** [comment]
Are there any hurdles wrt switching to postcss-8? E.g. `@vue/cli-service` still uses `css-loader` version 3 with `postcss` 7

**Comment by Uninen** [comment]
PostCSS 8 seems a bit impossible at the moment in Vue land where mostly everything uses 7.x. You can instead [use the compatibility build](https://tailwindcss.com/docs/installation#post-css-7-compatibility-build) which has identical features and still uses PostCSS 7: `tailwindcss@compat`

**Comment by forsartis** [comment]
New version released [v2.0.1](https://github.com/forsartis/vue-cli-plugin-tailwind/releases/tag/v2.0.1), this will install the compat version of Tailwind CSS. 
I also switched to Tailwind's internal PurgeCSS setup.

**Comment by Uninen** [comment]
Yay, thanks for the very fast response time and a great plugin 👍

**Comment by tylercollier** [comment]
Thanks! I was able to upgrade by removing the line `'vue-cli-plugin-tailwind/purgecss': {},` from my `postcss.config.js`'s `plugins` object (which I figured out by reading your updated README).

Just curious: would it be better to have autoprefixer and postcss as peer dependencies, [the same way that Tailwind 2 does](https://github.com/tailwindlabs/tailwindcss/blob/27fd1f9883997bb6cfde2bd6bde7981e23866ffb/package.json#L56-L57)?
